### PR TITLE
new game: Nexus (similar to The Game)

### PR DESCRIPTION
**Pull request by ArnoldSmith86**
Pretty simple game..

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: https://test.virtualtabletop.io/PR-1767/pr-test (or any other room on that server)

After merging, a backup will be available at `https://beta.virtualtabletop.io/editor/PR1767-pr-test`.

**Review by 96LawDawg — APPROVED** [review]
Added BGG Awards.  What's the point of the category if we don't use it?

Is there a reason you didn't use `hideWhenUnused` on the seats?  That seems to be the norm and looks better.  But it isn't necessary, so have an approval.

**Comment by ArnoldSmith86** [comment]
No, not on purpose. I'll change it like I did in the other games.

**Pull request merged**
